src/app/routines/publicRoutines/routineListItem/RoutineListItem.scss

```diff
@@ -19,4 +20,40 @@
     line-clamp: 2;
     -webkit-line-clamp: 2;
   }
+
+  &__likes {
+    width: 4.5rem;
+    height: 1.5rem;
+    display: flex;
+    border-radius: $border-radius-medium;
+    border: 1px solid $secondary-color;
```

Comment:
we have border variable that already has it preassigned (some style consistency within whole app)